src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $transitionSpeed;
```

Comment:
You should specify for which properties you want to apply the transition. 
Because currently it applies to all the properties and it's not good for optimization.
I guess Bohdan's comment was about that.